DataFormats/ForwardDetId/interface/MTDDetId.h

```diff
@@ -59,8 +59,10 @@ class MTDDetId : public DetId {
   /** Returns enumerated type specifying MTD sub-detector, i.e. BTL or ETL. */
   inline int mtdSubDetector() const { return (id_>>kMTDsubdOffset)&kMTDsubdMask; }
 
-  /** Returns MTD side, i.e. Z-=1 or Z+=2. */
+  /** Returns MTD side, i.e. Z-=1 or Z+=0. */
   inline int mtdSide() const { return (id_>>kZsideOffset)&kZsideMask; }
+  /** Return MTD side, i.e. Z-=-1 or Z+-1. */
+  inline int zside() const { return (id_>>kZsideOffset)&kZsideMask ? (1):(-1) ; }
```

Comment:
use `mtdSide()` in this function instead of duplicating the calculations

Reply:
ok